tmt/steps/prepare/ansible.py

```diff
@@ -168,25 +170,40 @@ class PrepareAnsible(tmt.steps.prepare.PreparePlugin[PrepareAnsibleData]):
 
     _data_class = PrepareAnsibleData
 
+    @property
+    def _preserved_workdir_members(self) -> set[str]:
+        return {
+            *super()._preserved_workdir_members,
+            # Include directories storing individual playbook logs.
+            *{str(i) for i in range(len(self.data.playbook))},
```

Comment:
These integer directory names are a bit confusing. Here's an example directory tree:

```
/var/tmp/tmt/run-232/plan/finish/
├── Ansible-we-want-to-test
│   └── 0
│       └── default-0
│           └── output.txt
```

Without reading the code I would not understand the extra levels and names. Not completely sure if it would be worth it but perhaps `playbook-0` and `guest-default-0` would make it clear. On the other hand, we probably do not want to make the paths tool long. So, just as a nitpick comment. Feel free to ignore.

Reply:
Correct, I forgot about that. Fixed in b6b71b46bbf2d7f8a295f018298162fb79873ec9